### ISSUE TITLE
error messages 

### ISSUE THREAD

**Issue by mikekaranja** [convoo/login-fire#88]
Is there anyway to get feedback on any error message from login-fire? for example in the properties such user, signIn, e.t.c

**Comment by sbeleidy** [comment]
There currently isn't a way to do that. We may potentially fire an event on failed sign in with the error messages but that will likely be in the 1.0 release we're currently working on and not on the previous releases.

Can you give us an example of how you want it to work or how you plan to use it so we can make sure we cover it?

**Comment by mikekaranja** [comment]
Hi thanks for the reply firing an event on failed sign in the firebase-auth.html file has worked for me so its ok thanks

**Comment by bkawk** [comment]
Are there any other cases other than the user has failed to sign in, that we are not handling?

I guess at the moment we know we are expecting the user details and if we don't get them we should wait for them until we feel like giving up, getting a fast response for a failed login makes sense to me. This may well fit in with the conversation we had with 3 attempts before requesting a captcha on strong passwords.

**Comment by sbeleidy** [comment]
Yeah this is directly related to #87. I think it's just the failed sign in that we're missing. This is also relevant in social if the user decides to cancel or if the login fails for whatever reason. Right now we don't do anything about that either. Maybe also a failed reset pw event? If the email doesn't exist in the db.

**Comment by debkanchan** [comment]
This is happening to me now.
Using login button gives a network error.
